packages/core/contracts/insured-bridge/implementation/L1_BridgeRouter.sol

```diff
@@ -0,0 +1,69 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity >=0.7.6;
+
+import "@eth-optimism/contracts/libraries/bridge/OVM_CrossDomainEnabled.sol";
+
+import "./L2_BridgeDepositBox.sol";
+
+contract BridgeRouter is OVM_CrossDomainEnabled {
+    address public l1Owner;
+    address public l2DepositBox;
+
+    event AddedL2DepositContract(address l2Contract);
+    event AddedL2DepositContract(address originToken, address destinationToken);
+    event L2DepositRelayed(
+        address sender,
+        address recipient,
+        address originToken,
+        address destinationToken,
+        address relayer,
+        uint256 amount,
+        uint256 realizedFee,
+        uint256 maxFee
+    );
+    event LPDepositCollateral(address token, uint256 amount, uint256 lpTokensMinted, address caller);
+    event L2TransferSpedUp(uint256 transactionId, address fastRelayer);
```

Comment:
```suggestion
    event L2TransferSpeedUp(uint256 transactionId, address fastRelayer);
```

nit: event name IMO should be SpeedUp instead of SpedUp